websites/F/Flowlab/presence.ts

```diff
@@ -0,0 +1,57 @@
+const presence = new Presence({ clientId: "991160367629750372" }),
+	flowlabTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "icon",
+			startTimestamp: flowlabTimestamp,
+		},
+		{ pathname, hostname } = document.location;
```

Comment:
`hostname` is only used once so shouldn't be a variable

Reply:
I changed it back to using `document.location.hostname` instead of the variable, but now I'm getting this error in the checks: `Error:   11:6  error  Use destructured variables over properties  unicorn/consistent-destructuring`

Reply:
Nevermind, you may still destrucutre `hostname`. I forgot about that ESLint rule

Reply:
Okay